ddtrace/tracer/tracer.go

```diff
@@ -768,7 +781,14 @@ func (t *tracer) Extract(carrier interface{}) (ddtrace.SpanContext, error) {
 	if !t.config.enabled.current {
 		return internal.NoopSpanContext{}, nil
 	}
-	return t.config.propagator.Extract(carrier)
+	ctx, err := t.config.propagator.Extract(carrier)
+	if t.config.tracingAsTransport {
+		// in tracing as transport mode, reset upstream sampling decision to make sure we keep 1 trace/minute
```

Comment:
```suggestion
		// in tracing as transport mode, reset upstream sampling decision to make sure we keep 1 trace/minute
		// TODO: replace with _dd.p.ts in the next iteration standardizing this for other products, comparing enabled products in `t.config` with their corresponding `_dd.p.ts` bitfields
```

Reply:
Applied in https://github.com/DataDog/dd-trace-go/pull/3033/commits/2be912cbe48a65d29cbee2f0699b41ef8e34e0bc